README.md

```diff
@@ -1 +1,13 @@
 # demo-weaver-for-dashboarding
+
+Generate all metrics and attributes used by them:
+
+```bash
+weaver registry generate  --registry=../semantic-conventions/model --templates=./templates go ./output
+```
+
+Generate metrics and relevant attributes in specific namespaces:
+
+```bash
+weaver registry generate  --registry=../semantic-conventions/model --templates=./templates go ./output --param included_namespaces="db,http"
+```
```

Comment:
Thanks, that's a lot better than my hardcoded go generate 😅 Maybe we could even delete `generate.go` in a separate PR. Or introduce a Makefile, not sure what's best. Anyway, work for follow-up PRs...

Reply:
thanks! I deleted `generate.go` now